main.go

```diff
@@ -4,8 +4,19 @@ Copyright © 2019 Kondukto
 */
 package main
 
-import "github.com/kondukto-io/kdt/cmd"
+import (
+	"fmt"
+	"os"
+
+	"github.com/kondukto-io/kdt/cmd"
+)
 
 func main() {
+	args := os.Args
+	if len(args) > 1 && args[1] == "version" {
+		fmt.Println("KDT Kondukto Client v1.0.7")
```

Comment:
`version` should be given from Makefile during compilation. 

```var Version = ""
...
fmt.Printf("KDT v%s", Version)
```

Reply:
Solved with b26a07a